### PR TITLE
Improve Controls Menu and apply a single notation format

### PR DESCRIPTION
**Pull request by Aminelouati**
In this PR, we:
1. Apply a single notation format for templates names in both Controls and Manuals Menu.
2. Transform Submit_Job_And_Wait workflow in a single task workflow.
3. Add 3 Jobs/Tasks (Start_PCA_Service, Execute_Action_PCA_Service, Finish_PCA_Service) in the "Controls" Menu of the Studio

### CODE REVIEW

File: app/templates/submit_job_and_wait.xml [hunk 1]
```diff
@@ -33,73 +32,46 @@
 
 // connect to the scheduler
 schedulerapi.connect()
+def jobid
+if(!variables.get("jobSubmitted")){
+    // get the workflow information from Task Variables
```
Comment (jrochas) [inline app/templates/submit_job_and_wait.xml]: Indentation could be better, but not that important...

File: app/templates/execute_action_PCA_service.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+def sessionId = schedulerapi.getSession()
+
+// Define PCA URL
+def schedulerRestUrl = variables.get("PA_SCHEDULER_REST_URL")
+def pcaUrl = schedulerRestUrl.replaceAll("/rest\\z", "/cloud-automation-service")
+
+// Connect to APIs
+def apiClient = new ApiClient()
+apiClient.setBasePath(pcaUrl)
+//apiClient.setDebugging(true)
```
Comment (luispinedamx) [inline app/templates/execute_action_PCA_service.xml]: is this comment kept as a hint for the user?

File: app/templates/finish_PCA_service.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+def sessionId = schedulerapi.getSession()
+
+// Define PCA URL
+def schedulerRestUrl = variables.get("PA_SCHEDULER_REST_URL")
+def pcaUrl = schedulerRestUrl.replaceAll("/rest\\z", "/cloud-automation-service")
+
+// Connect to APIs
+def apiClient = new ApiClient()
+apiClient.setBasePath(pcaUrl)
+//apiClient.setDebugging(true)
```
Comment (luispinedamx) [inline app/templates/finish_PCA_service.xml]: I guess same here

File: app/templates/start_PCA_service.xml [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+println("BEGIN " + variables.get("PA_TASK_NAME"))
+
+/*********************************************************************************
+* THIS PART IS SERVICE SPECIFIC. IF YOU NEED TO MODIFY SOMETHING, DO IT HERE       *
+/********************************************************************************/
+// It is possible to start a PCA service using some variables. In that case, we retreive their values in this part.  
+//e.g. def MY_Service_VAR_1 = $variables_WORKFLOW_VAR_1
+/********************************************************************************/
+
```
Comment (luispinedamx) [inline app/templates/start_PCA_service.xml]: I think these headers could be standardized, but it is just about aesthetics. Some lines starting with "*" others with "//" or "// "

Comment (Aminelouati) [reply app/templates/start_PCA_service.xml]: you are right @luispinedamx. I will harmonize

File: app/templates/execute_action_PCA_service.xml (hunk 2 above)
Comment (Aminelouati) [reply app/templates/execute_action_PCA_service.xml]: exactly

**Pull request merged**